packages/valory/services/oracle_hardhat/service.yaml

```diff
@@ -113,17 +106,15 @@ type: skill
         api_id: protocollabs3
     params:
       args: *id001
-    server_api:
-      args: *id002
     benchmark_tool:
-      args: *id003
+      args: *id002
 ---
 public_id: valory/ledger:0.19.0
 type: connection
 config:
   ledger_apis:
     ethereum:
-      address: ${SERVICE_ORACLE_HARDHAT_RPC:str:http://hardhat:8545}
```

Comment:
Can be only used with the dev mode

Reply:
No can also be used without dev mode. Check #52.